pkgs/development/tools/misc/elfutils/default.nix

```diff
@@ -48,7 +48,7 @@ stdenv.mkDerivation rec {
     sed -i s/run-backtrace-dwarf.sh//g tests/Makefile.in
   '';
 
-  outputs = [ "bin" "dev" "out" "man" ];
+  outputs = [ "bin" "dev" "out" "man" ] ++ lib.optional enableDebuginfod "libdebuginfod";
```

Comment:
I would suggest it to move to existing `$bin`, not the new `$libdebuginfo`. `bin/debuginfod` depends on `libdebuginfo.so` anyways. There seems to be no reason to keep a new snowflake derivation output if nothing uses it.

Reply:
most debug tools (gdb, perf, ...) depend on libdebuginfod.so but not on debuginfod itself (the lib is a client while debuginfod is a server implem), so moving debuginfod to $bin seems wasteful from a closure size point of view.